app/models/game.rb

```diff
@@ -49,6 +49,10 @@ def populate_board
     Rook.create(game_id: id, user_id: black_player_id, row: 7, col: 7, is_black: true)
   end
 
+  def blacks_turn?
+    !moves.last.piece.is_black
```

Comment:
This should have a check for if moves is empty (which would make `moves.last` nil)